### PR TITLE
General notes have been added.

### PR DESCRIPTION
**Pull request by Zkir**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,39 @@
...
+
+For S3 storage you can use Amazon services or any other cloud/hosting provider who provides S3 compartible storage.  For S3 storage creation please refer to [Amazon S3 documentation](https://docs.aws.amazon.com/AmazonS3/latest/userguide/GetStartedWithS3.html) or documentation of your hosting provider.
+
+For CKAN installation an configuration please refer to CKAN [Maintainer’s guide](https://docs.ckan.org/en/2.9/maintaining/index.html). 
+
+The following sections describe how to specify credentials for S3 storage and CKAN for the pipeline.
+### Setting up AWS credentials
 
-Since pipeline uploads produced datasets in S3 bucket you need to make sure:
 1. `AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY` and `AWS_REGION` are specified in environment variables. Please check this [manual](https://docs.aws.amazon.com/sdk-for-java/v1/developer-guide/setup-credentials.html) for more informarmation or use `aws configure`.
```
Comment (amurKontur) [inline README.md:30]: shouldn't we also mention that AWS output format should be specified too? For example, I used json. And it started going through datasets like ourairports and etc. it fell anyway, but firstly when I didn't have output format specified, it didn't come to this step

Comment (Zkir) [reply README.md:30]: I do not think it's needed. it should work without it.

File: README.md [hunk 2]
```diff
@@ -1,12 +1,56 @@
...
-3. In `config.inc.sh` variable `CKAN_DATA_S3_URL` should be specified to path in S3 where you expect datasets to be uploaded, for example `s3://geodata-eu-central-1-kontur-public/mapaction_dataset/` and variable `CKAN_DATA_URL` should point at the same path in S3 as `CKAN_DATA_S3_URL` but using http protocol, for example `https://geodata-eu-central-1-kontur-public.s3.amazonaws.com/mapaction_dataset/`.
\ No newline at end of file
+3. In `config.inc.sh` variable `CKAN_DATA_S3_URL` should be specified to path in S3 where you expect datasets to be uploaded, for example `s3://geodata-eu-central-1-kontur-public/mapaction_dataset/` and variable `CKAN_DATA_URL` should point at the same path in S3 as `CKAN_DATA_S3_URL` but using http protocol, for example `https://geodata-eu-central-1-kontur-public.s3.amazonaws.com/mapaction_dataset/`.
+
+Note: S3 paths should be specified with trailing slash /. Variable values should be specified WITHOUT quotes. 
+
+EXAMPLE
+
```
Comment (amurKontur) [inline README.md:41]: here add please a general phrase like 
# ckan integration

Comment (amurKontur) [reply README.md:41]: done

**Comment by amurKontur** [comment]
from me now it's ok

**Pull request merged**